doc/gui/examples/broadcast.py

```diff
@@ -59,9 +57,8 @@ def start_or_stop(state):
         thread_event.clear()
         thread = Thread(target=count, args=[thread_event, state.get_gui()])
         thread.start()
-    # Update button status.
-    # Because "button_text" is shared, all clients are updated
-    state.button_text = button_texts[1 if thread else 0]
+    # Update button status for all states.
+    broadcast_callback(state.get_gui(), update_state_var, ["button_text", button_texts[1 if thread else 0]])
```

Comment:
The legacy code (and comment) are appropriate and should not be changed:
we want to demonstrate that a shared variable (defined as such in `Gui.add_shared_variable("button_text")`) can be propagated to all clients in a single (and simple) line of code.